h/h_api/bulk_api/command_builder.py

```diff
@@ -53,16 +53,16 @@ class user:
         """User commands."""
 
         @classmethod
-        def upsert(cls, user_id, attributes):
+        def upsert(cls, attributes, id_reference):
```

Comment:
This is actually kind of nice. The user upsert is starting to look more like the group upsert. There could be some benefit to this when they are more similar.

Reply:
:+1: